Doc/library/stdtypes.rst

```diff
@@ -4181,6 +4181,12 @@ another set.  The :class:`frozenset` type is immutable and :term:`hashable` ---
 its contents cannot be altered after it is created; it can therefore be used as
 a dictionary key or as an element of another set.
 
+Note that as long as the objects have the same hash values, they are seen as
+equivalent by sets::
+
+   >>> {0, True, False, 1}
```

Comment:
Could you please add some of the types Terry listed in the issue too? I was frankly surprised at them (especially the Decimal and Fractions with the same hash value).
Maybe this would help:
```suggestion
   >>> {0, True, False, 1, 0.0, 1.0}
```